src/lib/discord/commando/events/message.js

```diff
@@ -2,6 +2,12 @@ module.exports = (client, msg) => {
 	// Ignore all bots
 	if (msg.author.bot) return
 
+	// echo everything send to bot to special debug channel
+	if (msg.channel.type === "dm" && client.config.discord.echoChannel !== "") {
+		client.channels.cache.get(client.config.discord.echoChannel).send("[<@"+msg.author+">] " + msg.cleanContent)
```

Comment:
We had to resolve an issue with the cache not being populated with a channel (see discordworker.js) - and had to move to fetch instead of cache.get as it wasn't always populated. And this could fail if the channel is wrong so I think exceptions need to be captured and logged